docs/routing/open-api.html

```diff
@@ -101,6 +101,8 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
       <main>
         <h1 class="heading-1">Kitura OpenAPI</h1>
         <p class="block-text">Kitura-OpenAPI is a library which makes it easy to add OpenAPI (aka Swagger) support to your Codable routing-based Kitura application.</p>
+        <p>In this guide we show you how easy it is to host both an automatically generated OpenAPI definition of your application on the endpoint `/openapi`and the SwaggerUI tool on the endpoint `/openapi/ui`. We also
```

Comment:
Missing space between `/openapi`and  - also would it read better with a comma? (quite a long sentence)

Reply:
@djones6 please can you rereview as I've implemented your review comments.